### ISSUE TITLE
Orientation of pose world landmarks coordinate system?

### ISSUE THREAD

**Issue by chubei** [google/mediapipe#3370]
Thank you for submitting a MediaPipe documentation issue.
The MediaPipe docs are open source! To get involved, read the documentation Contributor Guide
## URL(s) with the issue:

https://google.github.io/mediapipe/solutions/pose.html#pose_world_landmarks

## Description of issue (what needs changing):

The documentation only describes the origin of the coordinate system. I'm wondering what's the orientation?

**Comment by sureshdagooglecom** [comment]
Hi @chubei ,
Could you please explain use case w.r.t above isssue.

**Comment by chubei** [comment]
I'm trying to determine if the user is leaning forward or backward, and whether he/she is rotating the waist.

**Comment by sureshdagooglecom** [comment]
Hi @chubei ,
We can estimate the orientation of the pose skeleton by following the direction between the two shoulder landmarks or the two hip landmarks.

**Comment by chubei** [comment]
I don't think this solution fits our use case. Can you help confirm whether the orientation of the coordinate system is specified or not? If not, we'll probably look into other solutions.

**Comment by rm6759** [comment]
Older post but replying if anyone has similar question.
I prefer pose_landmarks instead of pose_world_landmarks.  I also use relative compass headings to make coding easier depending on how many keypoints and slopes I'm tracking.   

The camera is North.  x: 0.0 = East and x: 0.999 = West.  If x values for LSh and/or RSh are greater than LHi and/or RHi, the person is leaning to their left (West).  If LFt and LHi are greater (westof) than RFt and RHi, the person is oriented Northward.  I typically just use Above and Below for y:   The visibility value for each keypoint can be useful as well, however, visibility values can be high even if the keypoint is hidden, so I tend to use visibility as weight for calculating slopes.  

Finding the center point of Shoulders, Hips, and Ankles then drawing vertical lines from them down to the XY value of center between Ankles, a reasonable visual way to show if hips and shoulders are over center of gravity or leaning and to what extent.  

Python Ex:
def westof(lm1,lm2):
    if l1.x > l2.x:
        return True
    else:
        return False      

westof(LHi, LSh)

**Comment by kuaashish** [comment]
Hello @chubei,
We are upgrading the [MediaPipe Legacy Solutions](https://developers.google.com/mediapipe/solutions/guide#legacy) to [new MediaPipe solutions](https://developers.google.com/mediapipe/solutions) However, the libraries, documentation, and source code for all the MediapPipe Legacy Solutions will continue to be available in our [GitHub repository](https://github.com/google/mediapipe) and through library distribution services, such as [Maven](https://maven.google.com/web/index.html?#com.google.mediapipe) and [NPM](https://www.npmjs.com/search?q=%40mediapipe).

You can continue to use those legacy solutions in your applications if you choose. Though, we would request you to check [new MediaPipe solutions](https://developers.google.com/mediapipe/solutions) which can help you more easily build and customize ML solutions for your applications. These new solutions will provide a superset of capabilities available in the legacy solutions. Thank you

**Comment by chubei** [comment]
@kuaashish Thank you for your information. I'll check the new solutions and close this issue for now.
